src/adafruit_circuitplayground/constants.py

```diff
@@ -7,6 +7,8 @@
 
 BRIGHTNESS_RANGE_ERROR = "The brightness value should be a number between 0 and 1."
 
+CPX = "CPX"
```

Comment:
Putting it inside a device?

Reply:
like an enum?

Reply:
Based on my research, it seems that you usually don't put the constants in objects in python (https://stackoverflow.com/questions/6345840/whats-the-best-way-to-initialise-and-use-constants-across-python-classes). This was also how the last cohort arranged their constants.

Reply:
fell free to resolve this